openapi.yaml

```diff
@@ -476,7 +477,7 @@ paths:
                 "200":
                     content:
                         application/json:
-                            x-speakeasy-terraform-ignore: true
+                            x-speakeasy-ignore: true
```

Comment:
`terraform-ignore` makes it ignore a field from any CRUD API. speakeasy-ignore makes it ignore it only in this one spot. Let us make sure we're not not too stronghanded with ignoring things